fv3gfs-physics/fv3gfs/physics/stencils/physics.py

```diff
@@ -14,6 +14,8 @@
 from pace.dsl.typing import Float, FloatField
 from pace.util.grid import GridData
 
+from .._config import PhysicsConfig
```

Comment:
`from fv3gfs.physics import PhysicsConfig`

Reply:
Leaving as-is since this is an import from physics to physics.